source/js/widgets/right-pane.js

```diff
@@ -0,0 +1,288 @@
+                    text-shadow: 0 .2rem .9rem rgba(0,0,0,.5);
+                    border-top-left-radius: 10px;
+                    border-top-right-radius: 10px;
+                }
+
+                .month i{
+                    font-size: 2rem;
+                    cursor: pointer;     /* change ur cursor to a pointer*/
+                }
+
+                .month h1{
+                    font-size: 1.5rem;
+                    font-weight: 500;    /* boldness */
+                    text-transform: uppercase;
+                    letter-spacing: 0.2rem;
+                    margin-bottom: 0.2rem;
+                }
+
+                .month p{
+                    font-size: 1.2rem;
+                }
+
+                .weekdays{
+                    width: 100%;
+                    height: 3rem;
+                    margin-bottom: -0.3rem;
+                    padding: 0 0.4rem;
+                    display: flex;
+                    align-items: center;
+                }
+
+                .weekdays div{
+                    font-size: 1.2rem;
+                    font-weight: 400;
+                    letter-spacing: 0.1rem;
+                    width: calc(25rem / 7);
+                    display: flex;
+                    justify-content: center;
+                    align-items: center;
+                    text-shadow: 0 .3rem .5rem rgba(0,0,0,0.5rem);
+                }
+
+                .days{
+                    width: 100%;
+                    display: flex;
+                    flex-wrap: wrap;
+                    padding: 0.2rem;
+                }
+
+                .days div:not(.today){
+                    font-size: 1rem;
+                    margin: 0.2rem;
+                    width: calc(18.5rem / 7);
+                    height: 2.5rem;
+                    display: flex;
+                    justify-content: center;
+                    align-items: center;
+                    text-shadow: 0 .2rem .9rem rgba(0,0,0,.5);
+                    transition: background-color 0.2s;
+                    background-color: rgb(242, 242, 242);
+                    text-color: black;
+                }
+
+                /*if not today */
+                .days div:hover:not(.today){
+                    border: .2rem solid#777;
+                    cursor: pointer;
+                }
+
+                .prev-date,
+                .next-date{
+                    opacity: 0.5;
+
+                }
+
+                .today{
+                    background-color: #f39507;
+                    font-size: 1rem;
+                    margin: 0.2rem;
+                    width: calc(18.5rem / 7);
+                    height: 2.5rem;
+                    display: flex;
+                    justify-content: center;
+                    align-items: center;
+                    text-shadow: 0 .2rem .9rem rgba(0,0,0,.5);
+                    transition: background-color 0.2s;
+                    text-color: white;
```

Comment:
You can remove the 'text-color: white' ,black is fine and it is getting superseded by some parent class anyways atm.